html/components/award.stories.js

```diff
@@ -54,10 +54,9 @@ export const defaultStory = () => {
       class="sprk-o-Stack__item"
       data-sprk-toggle="container"
     >
-      <a
-        class="sprk-b-TypeBodyThree sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon"
+      <button
+        class="sprk-c-Toggle__trigger sprk-b-TypeBodyThree sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon"
```

Comment:
Does it need all these other classes? i thought RV was adding them to his new Sass code.

Reply:
RV's sass code is to strip out the user agent styles that browsers will add automatically now that this is a `button`. We don't want to change any of the styles that Spark was already applying.